src/Helper/Dsn.php

```diff
@@ -42,7 +42,7 @@ public function getDsn(): string
         $this->dsn = "$this->driver:" . "host=$this->host" . ';' . "dbname=$this->dbname";
 
         if ($this->port !== null) {
-            $this->dsn = $this->dsn . ';' . "port=$this->port";
+            $this->dsn .= ';' . "port=$this->port";
```

Comment:
`$this->dsn` can be replaced with local variable.

Reply:
Fixed.